test/inductor/test_torchinductor_codegen_dynamic_shapes.py

```diff
@@ -336,6 +337,7 @@ def run(*ex, **kwargs):
     # Refinement means we don't actually generate dynamic shapes (but only on
     # cpu apparently?!)
     "test_nonzero_unbacked_refinement_dynamic_shapes": TestFailure(("cpu",)),
+    **dynamic_shapes_test_failures,
```

Comment:
I guess the `expectedFailureCodegenDynamic` can also be removed from `test_AllenaiLongformerBase_repro`